app/forms/admin/schools/cohorts/appropriate_bodies/update_form.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Admin
+  module Schools
+    module Cohorts
+      module AppropriateBodies
+        class UpdateForm
+          include ActiveModel::Model
+          include ActiveRecord::AttributeAssignment
+          include ActiveModel::Serialization
+
+          TEACHING_SCHOOL_HUB_ID = "teaching_school_hub"
+
+          attr_accessor :appropriate_body_id, :teaching_school_hub_id, :school_cohort
+
+          validates :appropriate_body_id, presence: { message: I18n.t("errors.appropriate_body.blank") }
+          validates :teaching_school_hub_id, presence: { message: I18n.t("errors.appropriate_body.blank") }, if: :teaching_school_hub_selected?
```

Comment:
Could we use a slightly more specific error message for this one – "Select teaching school hub"?

Reply:
I've updated to be more specific